package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "atom-package-manager",
+  "name": "@atom-ide-community/atom-package-manager",
```

Comment:
Please update the `url` in `package.json` as well.

Reply:
Good catch.

Reply:
We don't have issues enabled here, so the "bugs" URL would point to nowhere in particular if we filled it out. Maybe I should point it to our "new pull request" URL...

Consider enabling issues here (I personally can't do that, apparently).

Reply:
When you link to https://github.com/atom-ide-community/apm/issues, and the issues aren't enabled for the repo, it _automatically_ redirects users to the ["pull requests" page](https://github.com/atom-ide-community/apm/pulls).

Kinda weird user experience, but it's much better than a 404, and it's what I was considering doing anyway. Other option is to delete the "bugs" object from the package.json entirely.